### DISCUSSION TITLE
Invalid URL pada VSC

### DISCUSSION THREAD

**Discussion by Githubmuls** [teamalgoritma/community#187] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107347991/178137602-55777bca-07b9-4780-a18b-8da2cbcc8f61.png)

Kendala ketika ingin melihat hasil VS Code (saat membuka file python app.py) dengan keterangan: invalid URL untuk sumber get datanya. Kira2 erornya dimana ya? Mohon pencerahan

**Comment by triarm** [comment, marked as answer]
Hi @Githubmuls ,
Dilihat dari error tersebut, sepertinya Anda belum menyimpan ulang (save) script app.py setelah melakukan pengeditan. Silahkan untuk lakukan saving file app.py tersebut kemudian jalankan kembali script app.py tersebut.

Semoga menjawab
